### DISCUSSION TITLE
How to show the virtual environment on the left side in (parentheses)?

### DISCUSSION THREAD

**Discussion by emoltz** [ohmyzsh/ohmyzsh#11285] [Help] [answered]
In the regular terminal, when I activate a virtual environment, it shows in the left hand side, like this:

`(venv) location $`

How do I enable this with omz?

**Comment by HwangTaehyun** [comment, marked as answer]
Hi, How about this?
https://stackoverflow.com/questions/38928717/virtualenv-name-not-show-in-zsh-prompt#:~:text=The%20following%20steps%20should%20solve%20the%20problem%3A

**Reply by carlosala** [reply]
This is the solution and works as expected!

**Reply by emoltz** [reply]
Thanks!

**Reply by RVasquez20** [reply]
Nice

**Reply by iavishnabs** [reply]
its good
